tests/fixtures/offline_store_feature_table/feature_cluster_two_features.json

```diff
@@ -140,7 +140,7 @@
                         "table_id": {
                             "$oid": "6337f9651050ee7d5980660d"
                         },
-                        "view_mode": "manual"
+                        "view_mode": "auto"
```

Comment:
Do you know why the view mode is different here? Are we using non-saved features to construct the feature cluster? If the feature is saved, the view mode should be changed to `manual`.

Reply:
Mixing calling API routes and services directly in service unit tests was causing some strange issues such as the deadlock between logging and alive_bar.

To prevent that, I made some changes to the test helper function `deploy_feature` to deploy a feature object using services instead of through the routes. That caused the change in view mode here, though the feature should be saved already:

https://github.com/featurebyte/featurebyte/blob/244466b36914b9a72a7b8bfa2f670ded3303879f/tests/util/helper.py#L625-L653